src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -1097,10 +1081,9 @@ public override TypeKind TypeKind
         {
             get
             {
-                // only classes and structs can have instance fields as tuple requires.
-                // we need to have some support for classes, but most common case will be struct
-                // in broken scenarios (ErrorType, Enum, Delegate, whatever..) we will just default to struct.
-                return _underlyingType.TypeKind == TypeKind.Class ? TypeKind.Class : TypeKind.Struct;
+                // From the language prospective tuple is a value type 
```

Comment:
fixed

---
In reply to: [81643451](https://github.com/dotnet/roslyn/pull/14161#discussion_r81643451) [](ancestors = 81643451)